vscode-dotnet-sdk-extension/webpack.config.js

```diff
@@ -45,7 +45,7 @@ const extensionConfig = {
   plugins: [
     new CopyPlugin({ patterns: [
       { from: path.resolve(__dirname, '../vscode-dotnet-runtime-library/install scripts'), to: path.resolve(__dirname, 'dist', 'install scripts') },
-      { from: path.resolve(__dirname, '../images'), to: path.resolve(__dirname, 'images') },
```

Comment:
Is removing the images purposeful?

Reply:
That was not intended, thank you so much for catching it 💦